db/migrate/20250910224538_add_sharing_fields_to_stats.rb

```
@@ -1,8 +1,17 @@
 # frozen_string_literal: true
 
 class AddSharingFieldsToStats < ActiveRecord::Migration[8.0]
-  def change
-    add_column :stats, :sharing_settings, :jsonb, default: {}
+  def up
+    add_column :stats, :sharing_settings, :jsonb
```

RuboCop (1.80.2)

[convention] You can use `change_table :stats, bulk: true` to combine alter queries.

(Rails/BulkChangeTable)